apps/judicial-system/web/src/components/CourtArrangements/CourtArrangements.tsx

```diff
@@ -8,7 +8,10 @@ import {
   DateTime,
 } from '@island.is/judicial-system-web/src/components'
 import { NotificationType } from '@island.is/judicial-system-web/src/graphql/schema'
-import { TempCase as Case } from '@island.is/judicial-system-web/src/types'
+import {
+  CourtDate,
```

Comment:
There is a generated type DateLog in the generated schema file that is identical to this interface. Please consider using that type instead.